daylay92/chat-app-boilerplate
# Add socket.io Setup

#### What does this PR do?
Add basic configuration socket.io configuration to server and client scripts.
#### Description of Task to be completed?
- Add the client `socket.io` script and configure the generic connect listener.
-  Implement a function for generating chat messages.
- Update `socket.js` file appropriately.
- Write unit tests for this feature
#### How should this be manually tested?
- Clone the repo and cd into the project folder.
- Checkout to `ch-setup-socket` branch and run `npm install`.
- Run `npm run start:dev` to spin-up the development server.
- Inspect the implementations described above.
- Run `npm test` to check out the unit test.
